src/app/components/generic-element/generic-element.component.ts

```diff
@@ -1,11 +1,12 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, ChangeDetectionStrategy } from '@angular/core';
 import { GenericElementData } from '../../models/parsed-elements';
 import { register } from '../../services/component-register.service';
 
 @Component({
   selector: 'evt-generic-element',
   templateUrl: './generic-element.component.html',
-  styleUrls: ['./generic-element.component.scss']
+  styleUrls: ['./generic-element.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
Why?

Reply:
It was an attempt to lighten the bindings and solve the problem of "freeze" on page changes. But we probably don't need it anymore